casper/src/test/scala/coop/rchain/casper/rholang/BlockRandomSeedSpec.scala

```diff
@@ -76,14 +77,16 @@ class BlockRandomSeedSpec extends AnyFlatSpec with Checkers {
     }
 
     // Reference values as hex strings
-    val seedHexRef =
-      "06414434353136000000000000000282080d420468760b7bf0dbe4aa5824a86198632646f24a0e651b9928ace27c3cf449ce2a2e3c875946511c83ba68dd0812034ba2b11842e15a3aab8110b4646068755a26ac647ffd7d2106d0e4cb2515a795a26ac647fbd7d2106d0e4cb2515a7230"
-    val rndHexRef = "0C338721fa06CAf348C9E013922E8B6D27C6E31FDC2B18FC6A83F144CA22D375"
+    val seedHexExpected =
+      "0641443435313601410406a102343b05bdf86df2552c125430cc319323792507328dcc9456713e1e7a24e715171e43aca3288e41dd346e840901a5d1588c2170ad1d55c0885a3230343aad1356323ffebe9083687265928ad3cad1356323fdebe9083687265928ad3918"
+    val rndHexExpected = "5d92fa794d3e68dcbdc0716de2b5799f95d04ef122109e21cb42f5fd7d64b359"
```

Comment:
Good question!

`BlockRandomSeedSpec.scala` was not in a subfolder and not executed on CI, so we didn't noticed it's failing.

Hex is changed because of the change in codec for seq. number.
https://github.com/rchain/rchain/commit/9490273aece539665a0088747c18443719aa463f#diff-8f2ec782815e751f46b8eaf76c90ad5540779f86e2a3b26becbc7e565dac389cR77